my_project/tab_psy_chart/app_psy_chart.py

```diff
@@ -33,10 +34,8 @@
     "None": "None",
     "Frequency": "Frequency",
 }
-psy_dropdown_names.update(dropdown_names.copy())
-psy_dropdown_names.update(sun_cloud_tab_dropdown_names.copy())
-psy_dropdown_names.update(more_variables_dropdown.copy())
-psy_dropdown_names.update(sun_cloud_tab_explore_dropdown_names.copy())
+psy_dropdown_names.update(psychrometric_chart_dropdown_names.copy())
```

Comment:
just use deep copy to be on the safe side

Reply:
please also check if in the code we are also using .copy() in other locations and replace that with deepcopy.